tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>instead of:</p><blockquote>The <code>Sampler</code> primitive can be used to sample the gradients of the quantum circuits. </blockquote><p>let's add more context:</p><p> </p><p>Following a similar logic to the estimator gradient, when we have a quantum state prepared by a quantum circuit, we can shift the parametrized gates by +- pi/2 and sample to compute the gradient of the sampling probability. </p><p> </p><pre class="ql-syntax" spellcheck="false">$$\frac{\partial}{\partial \theta} |\langle j | \psi(\theta)\rangle|^2 = (|\langle j | \psi(\theta + \pi/2)\rangle|^2 + |\langle j | \psi(\theta - \pi/2)\rangle|^2) / 2$$
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>